src/Compilers/CSharp/Test/Emit2/Semantics/ExtensionTypeTests.cs

```diff
@@ -10627,9 +10605,9 @@ public class Type
 """;
         var comp = CreateCompilation(src, targetFramework: TargetFramework.Net70);
         comp.VerifyDiagnostics(
-            // (1,8): error CS0117: 'object' does not contain a definition for 'Method'
-            // object.Method(); // PROTOTYPE revisit as part of "extension invocation" section
-            Diagnostic(ErrorCode.ERR_NoSuchMember, "Method").WithArguments("object", "Method").WithLocation(1, 8),
+            // (1,8): error CS0121: The call is ambiguous between the following methods or properties: 'E1.Method()' and 'E2.Method()'
+            // object.Method();
+            Diagnostic(ErrorCode.ERR_AmbigCall, "Method").WithArguments("E1.Method()", "E2.Method()").WithLocation(1, 8),
```

Comment:
How can one resolve this ambiguity? With casting?

Reply:
Those methods can be invoked on the extension types directly. Instead of `object.M()`, you'd do `E1.M()` or `E2.M()`.